lib/util/print.js

```diff
@@ -106,4 +107,46 @@ function printTreeTraversal(node, parentNode) {
   }
 }
 
+// Prints an expression node in LaTeX
+function printLatex(node, showPlusMinus=false) {
+  // Creates the LaTeX
+  function intoTex(node, showPlusMinus=false) {
+    const originalNodeTex = node.toTex();
+
+    let outputNodeTex;
+
+    if (!showPlusMinus) {
+      // Replaces '+-' with '-'
+      outputNodeTex = originalNodeTex.replace(/\s*?\+\s*?\-\s*?/g, ' - ');
+    }
+    else {
+      outputNodeTex = originalNodeTex;
+    }
+
+    const nodeSymbols = Symbols.getSymbolsInExpression(node);
+
+    // Creates the regex parts separately so that current symbol can be added
+    // in the middle
+    const regexFirstHalf = '\\s*?\\\\cdot\\s*?({?)\\s*?';
+    const regexSecondHalf = '\\s*?(}?)';
+
+    // Checks each symbol for explicit multiplication and removes it
```

Comment:
Hmmm don't think we need this stuff. More discussion in the general comment of this review. I'm open to discussion - let me know what you think!